src/ios/SafariViewController.m

```diff
@@ -30,26 +30,26 @@ - (void) show:(CDVInvokedUrlCommand*)command {
   bool readerMode = [options[@"enterReaderModeIfAvailable"] isEqual:[NSNumber numberWithBool:YES]];
   self.animated = [options[@"animated"] isEqual:[NSNumber numberWithBool:YES]];
   self.callbackId = command.callbackId;
-  
+
   vc = [[SFSafariViewController alloc] initWithURL:url entersReaderIfAvailable:readerMode];
   vc.delegate = self;
 
   bool hidden = [options[@"hidden"] isEqualToNumber:[NSNumber numberWithBool:YES]];
   if (hidden) {
     vc.view.userInteractionEnabled = NO;
-    vc.view.alpha = 0.0;
+    vc.view.alpha = 0.05;
```

Comment:
Lol, so giving it just a _bit_ of alpha _does_ work? I guess __0.01__ would work just as well? 😀